### PR TITLE
[tests-only] Make LDAP based test config more flexible for being able to run with libregraph/idm default config

### PR DESCRIPTION
**Pull request by rhafer**
## Description
This adds a couple of new LDAP related settings (via enviroment variables) in order to be able to run the LDAP based testsuite against the default libregraph-idm configuration as created by ocis (see: https://github.com/owncloud/ocis/pull/3331). Those settings can also be useful when testing ocis against different LDAP setups.

## Motivation and Context
To provide more flexibility when testing oCIS with and LDAP backend.

## How Has This Been Tested?
- manual run against oCIS

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

### CODE REVIEW

**Comment by phil-davis** [comment]
@rhafer GitHub claims that you have not signed the CLA yet? Please click the button...

**Review by phil-davis — COMMENTED** [review]
Looks good - only 1 suggestion.

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/core/34961/3/6
code-style found a couple of very minor spacing issues that need fixing.

File: tests/TestHelpers/OcisHelper.php [hunk 1]
```diff
@@ -232,7 +232,12 @@ public static function getLdapPort():int {
 	 * @return bool
 	 */
 	public static function useSsl():bool {
-		return (self::getLdapPort() === 636);
+		$useSsl = \getenv("REVA_LDAP_USESSL");
+		if ($useSsl === "") {
```
[the remark this answers is not in the document]
Comment (rhafer) [reply tests/TestHelpers/OcisHelper.php]: done

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIComments-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34974/129

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@rhafer feel free to merge if this is ready from your PoV.

**Comment by rhafer** [comment]
@phil-davis  Thanks. Will do. I still need to re-run this once more against reva, to make sure it doesn't break anything there.

**Pull request merged**